src/utilities/graphqlConnectionFactory.ts

```diff
@@ -1,6 +1,6 @@
 import { ConnectionPageInfo } from "../types/generatedGraphQLTypes";
 
-interface Interface_ConnectionEdge<T> {
+export interface Interface_ConnectionEdge<T> {
```

Comment:
If this export isn't needed, remove it, refer to comment above.

Reply:
Sure will do!